nns/nns_contract.go

```diff
@@ -336,17 +345,35 @@ func Register(name string, owner interop.Hash160, email string, refresh, retry,
 	} else {
 		updateTotalSupply(ctx, +1)
 	}
-	ns := NameState{
+	saveDomain(ctx, name, email, refresh, retry, expire, ttl, owner)
+	updateBalance(ctx, []byte(name), owner, +1)
+	postTransfer(oldOwner, owner, []byte(name), nil)
+	return true
+}
+
+//nolint:unused
```

Comment:
Remove this comment, add a comment about what this method does.

Reply:
removed, added